pkgs/by-name/do/dosbox/package.nix

```diff
@@ -16,6 +17,9 @@
   makeDesktopItem,
 }:
 
+let
+  sdl = if stdenv.hostPlatform.isDarwin then SDL else SDL_compat;
```

Comment:
`SDL` is currently an alias for `SDL_compat` so this conditional changes nothing.